src/store/epics/helpers/ChatHandler.js

```diff
@@ -195,7 +195,7 @@ class ChatHandler {
       this.ws.onmessage = this.onWsMessage;
       this.ws.onopen = (event) => {
         event.target.send(this.authToken);
-        observer.next();
+        observer.next(isNewRoom);
```

Comment:
If we emit this here, then `isNewRoom` is exposed to the place in the epic where we show "New room created". (Unintentionally borked previously.)